Sources/SwiftParser/Specifiers.swift

```diff
@@ -645,7 +684,9 @@ extension Parser {
       asyncSpecifier: asyncKeyword,
       RawUnexpectedNodesSyntax(unexpectedBeforeThrows, arena: self.arena),
       throwsSpecifier: throwsKeyword,
-      RawUnexpectedNodesSyntax(unexpectedAfterThrows, arena: self.arena),
+      nil,
```

Comment:
You don’t need to specify `nil` here.

Reply:
I do, because we're going through a protocol and there aren't default arguments there.

Reply:
Oh, sorry. I didn’t realize that when reading the code on GitHub 😉